jin_liaskos/week01/mta/js/mtaprogram.js

```diff
@@ -0,0 +1,96 @@
+// MTA homework
+// console.log() shows output similar to this:
+// "You must travel through the following stops on the N line: 34th, 28th, 23rd, Union Square."
+// "Change at Union Square."
+// "Your journey continues through the following stops: 23rd, 28th, 33rd."
+// "7 stops in total."
+// ================================================
+
+// query
+
+
+var lines = {
+  N: ['Times Square', '34th', '28th', '23rd', 'Union Square', '8th'],
+  L: ['8th', '6th', 'Union Square', '3rd', '1st'],
+	6: ['Grand Central', '33rd', '28th', '23rd', 'Union Square', 'Astor Place'],
+  // line: [],
+};
+
+// planTrip('N', 'Times Square', '6', '33rd');
+
+// declare objects/variables first object subway lines, stops
+
+//                             'N',  'Times Square',  '23rd'
+var singleLineTrip = function(line, startStation, endStation){
+  var stations = lines[line]; // ['Times Square', '34th', '28th', '23rd', 'Union Square', '8th']
+  console.log(stations);
+
+  var startIndex = stations.indexOf(startStation); // 0
+  var endIndex = stations.indexOf(endStation);  // 3
+  console.log(startIndex, endIndex);
+  var section = stations.slice(startIndex, endIndex);
```

Comment:
Nice work! But you still have to check that `startIndex < endIndex`, and if it's not, give those values to `slice()` in the reverse order, and reverse the array you get back.